raster-foundry/raster-foundry
# Upgrade GT Contrib and GT Server

## Overview

Upgrades GT Server and GT Contrib

### Checklist



## Testing Instructions

- Re-assemble everything, ensure that GDAL is enabled for backsplash (should be by default) in `docker-compose.yml`
- Open up QGIS and add http://localhost:8081/wms/410d4975-0eed-40c5-aa5a-511277507191 as a WMS layer
- See that 500s aren't thrown any longer

Closes #5019
